Core.Tests/TransactionTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FluentAssertions;
+using System;
+using Xunit;
+
+namespace Core.Tests
+{
+	public class TransactionTests
+	{
+		[Fact]
+		public void Constructor_TransactionIsLoaded_HashIsSet()
+		{
+			var transaction = new Transaction("1111", new Date(2000, 12, 31), new Money(1.99, Currency.UAH), "food", "Groceries");
+			transaction.GetHashCode().Should().NotBe(0);
+		}
+
+		[Fact]
+		public void Constructor_TransactionsDataAreSame_HashesAreEqual()
+		{
+			var transaction1 = new Transaction("1111", new Date(2000, 12, 31), new Money(1.99, Currency.UAH), "food", "Groceries");
+			var transaction2 = new Transaction("1111", new Date(2000, 12, 31), new Money(1.99, Currency.UAH), "food", "Groceries"); 
+			transaction1.GetHashCode().Should().Be(transaction2.GetHashCode());
+		}
+
+		[Fact]
+		public void Constructor_HashIsProvided_HashIsCorrect()
+		{
+			var expectedHash = 123;
```

Comment:
Codacy found an issue: [Add the 'const' modifier to 'expectedHash'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6066087)